content/04.discussion.md

```diff
@@ -16,34 +16,34 @@ We worked closely with pathologists and clinicians to build modules from which w
-An additional, large-scale cohort of >1,500 tumor samples and associated germline DNA is in the process of undergoing sequence data generation as part of CBTN CCDI-Kids First NCI and Common Fund project ([https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick](https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick)).
-Like the original OpenPBTA cohort, data will be processed and released in near real-time via the Kids First Data Resource and integrated with OpenPBTA.
-The OpenPBTA project has paved the way for new modes of collaborative data-driven discovery, open, reproducible, and scalable analyses that will extend beyond the current research described herein, and we anticipate this foundational work will continue to have a long-term impact within the pediatric brain tumor translational research community and beyond, ultimately leading to accelerated impact and improved outcomes for children with cancer.
+Another study harnessed OpenPBTA to integrate germline variants and discovered that pediatric HGG patients whose tumors undergo alternative lengthening of telomeres have enrichment of pathogenic or likely pathogenic germline variants in genes in the MMR pathway, oncogenic _ATRX_ mutations, and increased TMB [@doi:10.1093/neuonc/noac278].
+Moreover, OpenPBTA has enabled a framework to support real-time integration of clinical trial subjects as they enrolled on the PNOC008 high-grade glioma clinical trial [@clinicaltrials:NCT03739372] or PNOC027 medulloblastoma clinical trial [@clinicaltrials:NCT05057702], allowing researchers and clinicians to link tumor biology to translational impact through clinical decision support during tumor board discussions.
+Finally, as part of the the NCI's Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort ([https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis)) which enabled creation of the pediatric Molecular Targets Platform ([https://moleculartargets.ccdi.cancer.gov/](https://moleculartargets.ccdi.cancer.gov/)) in support of the RACE Act.
+An additional, large-scale cohort of >1,500 tumor samples and associated germline DNA is undergoing harmonization as part of CBTN CCDI-Kids First NCI and Common Fund project ([https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick](https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick)).
+Like the original OpenPBTA cohort, data will be processed and released in near real-time via the Kids First Data Resource and integrated with OpenPBTA data.
```

Comment:
> and integrated with OpenPBTA data.

This phrase reads to me like we will keep updating OpenPBTA data releases, as opposed to the other way around?

Reply:
this was an adam add ;) - let me think how to rephrase

Reply:
check [2917b01](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/443/commits/2917b01b577ffba399139f6722e04f4942de8fa3)